app/pages/SmartDisplay.tsx

```diff
@@ -19,13 +19,18 @@ function SmartDisplay() {
 
     return (
       <div className="main-container">
-        <Header stopDetails={stopDetails}/> 
-        <ul>
-          {departures.map((departure: any, index: number) => (
-            <DepartureRow key={index} departure={departure}/>
-            
-          ))}
-        </ul>
+        {
+          stopDetails
```

Comment:
So this means if the stopDetails call failed, you are not displaying the departures (they are 2 different calls, one can fail but the other might have gone through, you want to display as many avail data as possible as long as they can make sense on their own)

Reply:
In this case : I assume if no stop station no departure.